pn_counter_it_test.go

```diff
@@ -157,6 +157,12 @@ func TestPNCounter_SubtractAndGet(t *testing.T) {
 	})
 }
 
+func TestPNCounter_Reset(t *testing.T) {
+	it.PNCounterTester(t, func(t *testing.T, pn *hz.PNCounter) {
+		pn.Reset()
```

Comment:
Can we verify that reset is worked ? 
Increment the counter first. 
Close the instance to make sure that consistency is lost. 
Try an increment operation where an error should be returned.  `ConsistencyLostException` 
Then Reset the PnCounter.
Verify that increment is working without the exception.

Reply:
Updated the reset test and fixed PNCounter.